### PR TITLE
CUMULUS-1228 Use NGAP-compliant AMI

### PR DESCRIPTION
**Pull request by yjpa7145**

### CODE REVIEW

**Comment by yjpa7145** [comment]
This has been manually tested using both NGAP and non-NGAP AMIs, and with and without EFS configured.

File: CHANGELOG.md [hunk 1]
```diff
@@ -15,6 +15,12 @@ As a result of the changes for **CUMULUS-1193** and **CUMULUS-1264**, **you must
 
 ### BREAKING CHANGES
 
+- **CUMULUS-1228**
+  - The default AMI used by ECS instances is now an NGAP-compliant AMI. This
```
Comment (markdboyd) [inline CHANGELOG.md]: So does this mean that if you use a non-NGAP AMI, you can still deploy to a non-NGAP environment?

Comment (yjpa7145) [reply CHANGELOG.md]: Yes.  I added conditionals to the bootstrap script.

File: .gitignore [hunk 2]
```diff
@@ -31,6 +31,7 @@ search_index.json
 **/.tern-port
 /.tern-project
 /queries
+/example/app/efs-cf.yml
```
Comment (laurenfrederick) [inline .gitignore]: Is this for your branch or something that gets generated?

File: CHANGELOG.md (hunk 1 above)
Comment (laurenfrederick) [inline CHANGELOG.md]: So the default AMI, is the current (as in right now) NGAP AMI. Do we have documentation on how to find the AMI Id of the latest NGAP AMI?

I think in addition to having this info here, it should go in the deployment docs on this page: https://nasa.github.io/cumulus/docs/deployment/config_descriptions

File: .gitignore (hunk 2 above)
Comment (yjpa7145) [reply .gitignore]: That's a file that I have locally.  It's a CF template that will create an EFS volume that I can use for testing the EFS mount functionality in our ECS cluster.

File: CHANGELOG.md (hunk 1 above)
Comment (yjpa7145) [reply CHANGELOG.md]: Updated in d9eebbb57

**Pull request merged**